README.md

```diff
@@ -1171,80 +1171,113 @@ Wikipedia says
 
 Translating our account example above. First of all we have a base account having the logic for chaining the accounts together and some accounts
 
-```php
-abstract class Account {
-    protected $successor;
-    protected $balance;
-
-    public function setNext(Account $account) {
-        $this->successor = $account;
-    }
-    
-    public function pay(float $amountToPay) {
-        if ($this->canPay($amountToPay)) {
-            echo sprintf('Paid %s using %s' . PHP_EOL, $amount, get_called_class());
-        } else if ($this->successor) {
-            echo sprintf('Cannot pay using %s. Proceeding ..' . PHP_EOL, get_called_class());
-            $this->successor->pay($amountToPay);
-        } else {
-            throw Exception('None of the accounts have enough balance');
+```c#
+    /// <summary>
```

Comment:
I agree, code should be the most simple possible and follow only the required language conventions.